python/ray/tests/test_basic.py

```diff
@@ -28,28 +28,18 @@
 import ray
 import ray.tests.cluster_utils
 import ray.tests.utils
+from ray.tests.fixtures import (
+    ray_start_2_cpus,
+    ray_start_cluster,
+    ray_start_regular,
+    shutdown_only,
+)  # noqa: F401,F811
```

Comment:
What's the linting error here? We shouldn't need the `noqa`. Same comment applies in other places.

Reply:
The lint error looks strange to me. May the usage is not correct here.
```
./python/ray/tests/test_basic.py:31:1: F401 'ray.tests.fixtures.ray_start_2_cpus' imported but unused
./python/ray/tests/test_basic.py:31:1: F401 'ray.tests.fixtures.ray_start_cluster' imported but unused
./python/ray/tests/test_basic.py:31:1: F401 'ray.tests.fixtures.ray_start_regular' imported but unused
./python/ray/tests/test_basic.py:31:1: F401 'ray.tests.fixtures.shutdown_only' imported but unused
./python/ray/tests/test_basic.py:42:31: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:109:32: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:312:32: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:340:52: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:372:52: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:387:18: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:415:29: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:445:39: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:462:25: F811 redefinition of unused 'ray_start_2_cpus' from line 31
./python/ray/tests/test_basic.py:605:23: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:690:34: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:747:36: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:815:21: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:873:23: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:884:36: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:894:19: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:903:15: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:958:25: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:979:34: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:1010:35: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:1053:42: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:1080:24: F811 redefinition of unused 'ray_start_2_cpus' from line 31
./python/ray/tests/test_basic.py:1124:23: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:1150:31: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:1191:35: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:1257:28: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:1272:25: F811 redefinition of unused 'ray_start_2_cpus' from line 31
./python/ray/tests/test_basic.py:1409:34: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:1496:21: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:1579:31: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:1655:37: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:1708:18: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:1861:20: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:1872:26: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:1890:31: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:1941:36: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:2067:27: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:2103:31: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:2159:32: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2234:25: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:2269:25: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:2307:25: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:2326:43: F811 redefinition of unused 'ray_start_cluster' from line 31
./python/ray/tests/test_basic.py:2370:27: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2508:28: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2531:32: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2556:18: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2633:30: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2649:27: F811 redefinition of unused 'ray_start_2_cpus' from line 31
./python/ray/tests/test_basic.py:2668:35: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2694:20: F811 redefinition of unused 'ray_start_2_cpus' from line 31
./python/ray/tests/test_basic.py:2742:34: F811 redefinition of unused 'shutdown_only' from line 31
./python/ray/tests/test_basic.py:2750:46: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:2771:28: F811 redefinition of unused 'ray_start_regular' from line 31
./python/ray/tests/test_basic.py:2819:31: F811 redefinition of unused 'shutdown_only' from line 31
```

Reply:
I checked pytest document. I need to move the fixtures from `fixtures.py` to `conftest.py`. Then we can use all the fixtures without imports.